### DISCUSSION TITLE
Vertical branches in Gitgraph

### DISCUSSION THREAD

**Discussion by DamienCassou** [mermaid-js/mermaid#3772] [Ideas]
Currently, Gitgraph always represents branches horizontally:
![Screenshot 2022-11-07 at 21-05-52 Gitgraph (Git) Diagram 🔥](https://user-images.githubusercontent.com/217543/200404409-7ef13c18-5645-4258-bddb-21eee81168bd.png)

It would be nice to be able to render them vertically as well such as in this mockup:
![Screenshot 2022-11-07 at 21-00-30 Gitgraph (Git) Diagram 🔥](https://user-images.githubusercontent.com/217543/200404397-32f5373a-03f2-4fda-a74f-fd8334279f0b.png)

**Comment by sjackson0109** [comment]
Dude, thats what I said!
https://github.com/mermaid-js/mermaid/issues/2585

**Reply by meowcat** [reply]
Upvote, this would be great

**Reply by redzioch** [reply]
+1
